db/changes.go

```diff
@@ -241,6 +246,11 @@ func (db *DatabaseCollectionWithUser) buildRevokedFeed(ctx context.Context, ch c
 
 			sentChanges := 0
 			for _, logEntry := range changes {
+				if options.ChangesCtx.Err() != nil {
```

Comment:
If we're checking here for each loop iteration, I don't think we need to check again on line 307 - what do you think?